exchanges/exchange_test.go

```diff
@@ -208,7 +208,7 @@ func TestSetClientProxyAddress(t *testing.T) {
 		t.Error("SetClientProxyAddress parsed invalid URL")
 	}
 
-	if newBase.Websocket.GetProxyAddress() != "" {
+	if newBase.Websocket.GetProxyAddress().String() != "" {
```

Comment:
Think that this would work with assert.NotEmpty
And we should be explicit about why we're expecting this, e.g. "SetClientProxyAddress should not set ProxyAddress on error"